core/src/test/java/org/infinispan/distribution/topologyaware/TopologyAwareConsistentHashFactoryTest.java

```diff
@@ -62,45 +65,50 @@ public void testNumberOfOwners() {
       addNode(testAddresses[0], "m0", null, null);
 
       updateConsistentHash(1);
-      assertEquals(ch.locateOwners(testAddresses[0]).size(), 1);
+      assertNumberOwners(testAddresses[0], 1);
       updateConsistentHash(2);
-      assertEquals(ch.locateOwners(testAddresses[0]).size(), 1);
+      assertNumberOwners(testAddresses[0], 1);
 
       addNode(testAddresses[1], "m1", null, null);
 
       updateConsistentHash(1);
-      for (Address testAddress : testAddresses) {
-         assertEquals(ch.locateOwners(testAddress).size(), 1);
+      for (Address testAddress : ch.getMembers()) {
```

Comment:
This is kind of outdated, the test should iterate over the segments and check the owners of each segment instead.